src/simulation/sensors/camera/Custom.cmake

```diff
@@ -1,14 +1,9 @@
-if(BUILD_OPNAV)
-  #  find_package(opencv CONFIG REQUIRED)
-  #
-  #  set(CUSTOM_CMAKE_BUILD_TARGETS opencv::opencv)
-  #
-  #  set(CMAKE_INCLUDE_CURRENT_DIR TRUE)
-  #  if(APPLE)
-  #    link_libraries("-framework OpenCL")
-  #  endif()
-  include(usingOpenCV)
-else()
-  message("SKIPPED: ${TARGET_NAME}")
-  set(CUSTOM_DEPENDENCIES_HANDLED 1)
-endif()
+#  find_package(opencv CONFIG REQUIRED)
+#
+#  set(CUSTOM_CMAKE_BUILD_TARGETS opencv::opencv)
+#
+#  set(CMAKE_INCLUDE_CURRENT_DIR TRUE)
+#  if(APPLE)
+#    link_libraries("-framework OpenCL")
+#  endif()
```

Comment:
Perhaps you forgot to remove these comments?

Reply:
Thanks! Removed.